docs/install.rst

```diff
@@ -1,15 +1,70 @@
 Installation
 ============
 
-This installation guide includes only the GeoCAT-examples installation instructions.
-Please refer to `GeoCAT Contributor's Guide <https://geocat.ucar.edu/pages/contributing.html>`_
-for installation of the whole GeoCAT project.
+GeoCAT-examples is a collection of standalone visualization examples, and it is not distributed
+as a conda package; thus, there is no installation for it (via conda or other package management systems).
+The easiest way to access GeoCAT-examples scripts as a whole is by cloning the repo and then using a
+`Conda <http://conda.pydata.org/docs/>`_ environment.
 
-Installing GeoCAT-examples
---------------------------
+For information about installing the entire GeoCAT project, see our
+[Contributor's Guide](https://geocat.ucar.edu/pages/contributing.html#3-creating-a-development-environment).
 
-GeoCAT-examples is a collection of standalone visualization examples, and it is not distributed
-as a conda package; thus, there is no installation for it (via conda or other package management
-systems). Because of this, this sections refers to building GeoCAT-examples, and it can be followed
-better through the `Installation document on Github repository
-<https://github.com/NCAR/GeoCAT-examples/blob/main/INSTALLATION.md>`_.
+Creating a Conda environment
+----------------------------
+
+The GeoCAT-examples repository has two different types of implementation for visualization examples:
+
+1.  Python scripts (:code:`.py`) under the "Gallery" and "GeoCAT-comp-examples" directories
```

Comment:
This actually applies to more than just this PR, but is there an advantage to formatting with `:code:` vs this?:
```suggestion
1.  Python scripts (``.py``) under the "Gallery" and "GeoCAT-comp-examples" directories
```

Reply:
`:code:` is the only way I had seen it done before, so I copied the formatting. I'm not sure if there is a difference. @erogluorhan do you know?

Reply:
Sphinx renders both `:code:` and double backquotes as code AFAIK, but there might still be a functional difference between two in different renderers as they syntactically different.

In our docs, `:code:` was used to differ some texts (even though they were not actually code snippets) in the past. 

I think what Anissa suggests here makes sense to me.

Reply:
I'll go ahead and change this to use double backquotes instead for inline mentions of command names, package names, arguments, etc.